client/src/components/AddNewProfile.js

```diff
@@ -212,6 +227,18 @@ const AddNewProfile = () => {
               }
             />
           </div>
+          <div class="sm:col-span-6">
+            <label className="font-semibold text-gray-700" htmlFor="groups">
+              Language
+            </label>
+            <Select
+              id="groups"
+              name="groups"
+              placeholder="Select language ..."
+              options={languageOptions}
+              onChange={selected => updateField("languages", selected.value)}
+            />
+          </div>
```

Comment:
The code here is still referring to the `groups` setting: `htmlFor`, `id`, `name`.